sql/core/src/test/resources/sql-tests/inputs/subquery/in-subquery/in-nulls.sql

```diff
@@ -0,0 +1,15 @@
+create table t0(t0a int not null, t0b int) using parquet;
```

Comment:
we usually create temp views in the test, as tables are persistent and may affect other test cases.

Reply:
How do I define a column as `not null` in a temp view? Is it possible?

If not, I can drop the tables at the end of the test.

Reply:
temp view infers nullability from the input query
```
scala> sql("create temp view v as select 1 as non_nullable, cast(null as int) as nullable")
res2: org.apache.spark.sql.DataFrame = []

scala> spark.table("v").printSchema
root
 |-- non_nullable: integer (nullable = false)
 |-- nullable: integer (nullable = true)
```

BTW, parquet table can't have non-nullable columns in Spark, so this test doesn't actually work.

Reply:
Thanks. I had been trying to create temp view with `values` and it was always nullable, but this works.